### PR TITLE
Fix/149541 Update URLs in Github templates

### PR DESCRIPTION
**Pull request by catleeball**
# Creating the pull request

> Please include a summary of the change and which issue is fixed\
> If the related issue does not exist in our repository, please create it before making pull request\
> It is highly recommended to use our [Web Reporting Tool](https://kb.adguard.com/en/technical-support/reporting-tool) instead of creating an issue on GitHub directly\
> Please note, that we verify every pull request manually, so it may take time to apply it

### To avoid invalid pull requests, please check and confirm following terms

- [x] This is not an ad/bug report;
- [x] My code follows the [guidelines](https://github.com/AdguardTeam/AdguardFilters/blob/master/CONTRIBUTING.md) and [syntax](https://kb.adguard.com/general/how-to-create-your-own-ad-filters) of this project;
- [x] I have performed a self-review of my own changes;
- [x] My changes do not break web sites, apps and files structure.

### If the problem does not fall under any category that is listed here, please write a comment below in corresponding section

Fix a broken URL to the reporting tool in the Github templates.

### Enter the issue address

- https://github.com/AdguardTeam/AdguardFilters/issues/149541

### Terms

- [x] By submitting this issue, I agree that pull request does not contain private info and all conditions are met

### CODE REVIEW

File: SpywareFilter/sections/tracking_servers.txt [hunk 1]
```diff
@@ -3316,6 +3316,8 @@
 ||signup-way.com^$third-party
 ||silverpop.com^$third-party
 ||silverpush.co^$third-party
+||simpleanalyticscdn.com^$third-party
```
Comment (Sergey-Lyapin) [inline SpywareFilter/sections/tracking_servers.txt:3319]: Hello @catleeball
We already have this rule in tracking_servers.txt

Comment (krystian3w) [reply SpywareFilter/sections/tracking_servers.txt:3319]: Bad day #149538

File: SpywareFilter/sections/tracking_servers.txt [hunk 2]
```diff
@@ -3316,6 +3316,8 @@
...
+||simpleanalyticsexternal.com^$third-party
```
Comment (krystian3w) [inline SpywareFilter/sections/tracking_servers.txt:3320]:
```suggestion
```

File: SpywareFilter/sections/tracking_servers.txt (hunk 1 above)
Comment (krystian3w) [inline SpywareFilter/sections/tracking_servers.txt:3319]:
```suggestion
```

**Review by krystian3w — CHANGES_REQUESTED** [review]

**Review by krystian3w — APPROVED** [review]

File: SpywareFilter/sections/tracking_servers.txt (hunk 1 above)
Comment (krystian3w) [reply SpywareFilter/sections/tracking_servers.txt]: It is a pity that Microsoft did not add to this system the option to undo two lines in one commit as Sergei's comment is not a problem in this case.

**Review by Sergey-Lyapin — APPROVED** [review]

**Pull request merged**